### PR TITLE
Add Regression Models

### PR DESCRIPTION
**Pull request by qur786**
@maneetgoyal sir , for now I am just trying to console log these things, but this is not creating the js files.

### CODE REVIEW

File: src/models/regression/index.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
+import fs from "fs";
```
Comment (maneetgoyal) [inline src/models/regression/index.ts:1]: Don't use `fs`. the libs you use should be browser friendly. `fs` may not run on browser. it's a nodejs lib.

So use a different lib.

File: src/models/regression/index.ts [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 import { RandomForestRegression } from "ml-random-forest";
 import MultivariateLinearRegression from "ml-regression-multivariate-linear";
 import { RobustPolynomialRegression } from "ml-regression-robust-polynomial";
 import { kombi } from "kombi";
```
Comment (maneetgoyal) [inline src/models/regression/index.ts:5]: `kombi` is imported but not used. refactor code accordingly.

**Review by maneetgoyal — CHANGES_REQUESTED** [review]
will pull the branch in a bit

**Comment by qur786** [comment]
Yeah @maneetgoyal sir the above issues are okay, but I have tried it with simple regression modules and used simple arrays as well, but it was not creating the js file on build. Can you please make some changes according to the browser implementation and try to build the js file and check if the contents of `models/regression/index.ts` is there in the build file or not?

**Comment by maneetgoyal** [comment]
>Can you please make some changes according to the browser implementation and try to build the js file and check if the contents of models/regression/index.ts is there in the build file or not?

Already mentioned that I will pull the branch in a bit. have some patience!

**Comment by maneetgoyal** [comment]
changes made, regression content sis now in final js bundle

**Review by maneetgoyal — APPROVED** [review]

**Comment by maneetgoyal** [comment]
demo can be made via separate PR, these changes can go in

**Pull request merged**